Lombiq.OrchardCoreApiClient/Models/TenantSetupApiModel.cs

```diff
@@ -23,6 +26,21 @@ public class TenantSetupApiModel
     [DataType(DataType.Password)]
     public string Password { get; set; }
     public string RecipeName { get; set; }
-    public JToken Recipe { get; set; }
+
+    [Obsolete($"This property relies on a deprecated NuGet package and will be removed in the next major version. " +
+              $"Please use the {nameof(RecipeJson)} property instead and serialize the value yourself in a project " +
+              $"that doesn't use the .NET Standard framework.")]
+    [JsonIgnore]
+    public IFormFile Recipe
+    {
+        get => RecipeJson.ToObject<IFormFile>();
+        set => JToken.FromObject(value);
+    }
+
+#pragma warning disable CS0618 // Use of Obsolete symbol. But only for reference.
+    [JsonProperty(nameof(Recipe))]
```

Comment:
Use a string name instead, because this should keep working when the other property is removed.